EdFi.Buzz.Database/migrations/sqls/20201201170923-alter-studentasssessment-up.sql

```diff
@@ -3,6 +3,9 @@
 -- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 -- See the LICENSE and NOTICES files in the project root for more information.
 
+DROP EXTENSION pgcrypto CASCADE;
```

Comment:
I got an error in this line (I think): 
`ifError got unwanted exception: extension "pgcrypto" does not exist`

I wonder if we can add something like Drop it if it exists.

Reply:
If we can use cascade, surely we can. I will check!